extension/chrome/elements/compose-modules/compose-render-module.ts

```diff
@@ -98,9 +98,11 @@ export class ComposeRenderModule extends ViewModule<ComposeView> {
       const thread = await this.view.emailProvider.threadGet(this.view.threadId, 'metadata');
       const inReplyToMessage = thread.messages?.find(message => message.id === this.view.replyMsgId);
       if (inReplyToMessage) {
-        this.view.replyParams.inReplyTo = inReplyToMessage.payload?.headers?.find(
-          header => header.name === 'Message-Id' || header.name === 'Message-ID'
-        )?.value;
+        const msgId = inReplyToMessage.payload?.headers?.find(header => header.name === 'Message-Id' || header.name === 'Message-ID')?.value;
+        const references = inReplyToMessage.payload?.headers?.find(header => header.name === 'References')?.value;
+        if (msgId && references) {
+          this.setReplyHeaders(msgId, references);
+        }
```

Comment:
no need to check `if (msgId && references)`, as they both are optional parameters for `setReplyHeaders`, and we have check for `if (msgId)` inside `setReplyHeaders`

Reply:
got it, I've removed the unneeded checks.